app/coffee/Features/Project/ProjectController.coffee

```diff
@@ -186,7 +195,7 @@ module.exports = ProjectController =
 
 		async.parallel {
 			project: (cb)->
-				Project.findPopulatedById project_id, cb
+				Project.findOne { _id: project_id }, cb
```

Comment:
This can become 

	ProjectGetter.getProject project_id, {lastUpdated:true, name:true}